### ISSUE TITLE
Is encrypting file paths out of scope now?

### ISSUE THREAD

**Issue by kdenhartog** [decentralized-identity/identity-hub#82]
Sorry if I'm rehashing a question that's already been resolved, I've not been following this work closely for awhile now. I just saw some discussion that raised the question in my mind from #76 and was wondering if file paths are going to be encrypted from the host or if this has been placed out of scope. By "host" I mean the computer that's operating the identity-hub software. So for example if I'm running my identity-hub in the cloud, do I need to trust my cloud provider in knowing which objects I have stored even if they can't see the data of the objects.

**Comment by csuwildcat** [comment]
This does not appear to be an issue given the current spec/assumptions. Reopen a new issue if you see anything that needs to be addressed.

**Comment by kdenhartog** [comment]
Yeah, I've stopped following a lot of this work so thanks for closing @csuwildcat

Best of luck on continuing to move this stuff forward!
